iac/aws/batch/modules/storage/scripts/table_setup.sql

```diff
@@ -1,7 +1,3 @@
--- DROP DATABASE ailab_tracker;
```

Comment:
these statements are being removed because the lambda function is directly connecting to the database that is created using Terrraform